src/QUESTIONS.md

```diff
@@ -0,0 +1,426 @@
+## General
+
+### What CHERI is?
+
+See
+[An Introduction to CHERI](https://www.cl.cam.ac.uk/techreports/UCAM-CL-TR-941.pdf).
+
+
+### What CHERI-extended hardware is available to use?
+
+[Arm Morello](https://www.arm.com/architecture/cpu/morello)
+is currently the only System-on-Chip with a CHERI-extended CPU.
+
+There are several
+[CHERI-RISC-V prototypes on FPGA](https://www.cl.cam.ac.uk/research/security/ctsrd/cheri/cheri-risc-v.html),
+including a superscalar, out-of-order CHERI-Toooba core extending
+[Toooba](https://github.com/bluespec/Toooba)
+that
+is based on
+[RiscyOO](https://github.com/csail-csg/riscy-OOO).
+
+Additionally, CHERI-RISC-V and Morello can be emulated using QEMU-CHERI or
+Morello FVP.
+
+
+### How can I get an Arm Morello board?
+
+Go to the
+[Digital Security by Design](https://www.dsbd.tech/get-involved/morello-board-request/)
+website and follow the instructions there.
+
+
+### How can I emulate CHERI?
+
+1. Clone the
+   [cheribuild](https://github.com/CTSRD-CHERI/cheribuild)
+   repository.
+
+1. [Install dependencies](https://github.com/CTSRD-CHERI/cheribuild#pre-build-setup)
+   for your operating system.
+
+1. Build a software stack and run CheriBSD in a QEMU-based VM.
+
+   * For Arm Morello: `cheribuild.py run-morello-purecap -d`
+
+   * For CHERI-RISC-V: `cheribuild.py run-riscv64-purecap -d`
+
+
+## Operating Systems
+
+
+### What operating systems are being developed for CHERI?
+
+* [CheriBSD](https://cheribsd.org/) for
+  [Arm Morello](https://www.arm.com/architecture/cpu/morello) and
+  [CHERI-RISC-V](https://www.cl.cam.ac.uk/research/security/ctsrd/cheri/cheri-risc-v.html)
+
+* [Linux](https://linux.morello-project.org/docs/) for
+  [Arm Morello](https://www.arm.com/architecture/cpu/morello)
+
+* [Android](https://git.morello-project.org/morello/docs/-/blob/morello/mainline/android/readme.rst)
+  for
+  [Arm Morello](https://www.arm.com/architecture/cpu/morello)
+
+* [CHERIoT RTOS](https://github.com/microsoft/cheriot-rtos) for the
+  [CHERIoT platform](https://aka.ms/cheriot-tech-report)
+
+* seL4
+
+  There are several institutions interested in developing seL4 for CHERI.
+  You can talk to them in the `#sel4` channel at the
+  [CHERI-CPU](https://www.cl.cam.ac.uk/research/security/ctsrd/cheri/cheri-slack.html)
+  Slack.
+
+
+### Which operating system should I use to experiment with CHERI?
+
```

Comment:
At the moment CheriBSD is the most fully featured operating system running on CPUs with CHERI extensions.  If you intend to work with code that runs on UNIX-like operating systems we recommend you start with CheriBSD.

Reply:
I've rephrased that to merge with previous text.